std/uni.d

```diff
@@ -7094,11 +7103,34 @@ int icmp(S1, S2)(S1 str1, S2 str2)
 }
 
 ///
-unittest{
+unittest
+{
     assert(icmp("Rußland", "Russland") == 0);
     assert(icmp("ᾩ -> \u1F70\u03B9", "\u1F61\u03B9 -> ᾲ") == 0);
 }
 
+/**
+ * By using $(XREF utf, byUTF) and its aliases, GC allocations via auto-decoding
+ * and thrown exceptions can be avoided.
+ */
+@safe nothrow pure unittest
```

Comment:
Huh, I guess you can add it. I thought that the rules about array literals applied to string literals.